float32 X
float32 Y
float32 Z
float32 forward_vel
float32 sway_vel
float64 lat
float64 long
float32 altitude
int8 ValidGPSfix
float32 th_rpm_fh # estimated rpm of front thruster
float32 th_rpm_ah # estimated rpm of rear thruster

